galaxy_ng/app/api/ui/viewsets/my_namespace.py

```diff
@@ -0,0 +1,18 @@
+
+from galaxy_ng.app import models
+from galaxy_ng.app.api import permissions
+from galaxy_ng.app.api.v3.viewsets.namespace import NamespaceViewSet
+
+
+class MyNamespaceViewSet(NamespaceViewSet):
```

Comment:
The UI expects groups in the my namespaces to use the old serialization where groups are just a list of names.

Reply:
Yeah, this should be subclassing ui.viewsets.namespaces.NamespaceViewset instead. Will update.

Reply:
@newswangerd Updated.